ublox/src/ubx_packets/packets.rs

```diff
@@ -2013,6 +2014,21 @@ struct EsfIns {
     z_accel: i32,
 }
 
+#[ubx_extend_bitflags]
+#[ubx(from, rest_reserved)]
+bitflags! {
+    pub struct EsfInsBitFlags: u32 {
+        const VERSION = 1;
+        const X_ANG_RATE_VALID = 0x100;
```

Comment:
How does this differentiate between version being 8 bits and the rest of the flags being 1 bit?